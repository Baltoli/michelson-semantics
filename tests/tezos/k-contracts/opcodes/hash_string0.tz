$contract { parameter string;
storage bytes;
code {CAR; PACK ; BLAKE2B; NIL operation; PAIR};
 } ;
$param "abcdefg" ;
$storage 0x00 ;
$ExpectedOutput 0x46fdbcb4ea4eadad5615cdaa17d67f783e01e21149ce2b27de497600b4cd8f4e ;

