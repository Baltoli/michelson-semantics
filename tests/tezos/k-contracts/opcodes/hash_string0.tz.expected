<generatedTop>
  <k>
    BLAKE2B .AnnotationList ~> NIL .AnnotationList operation .AnnotationList ; PAIR .AnnotationList ~> #TransformExpected ( 0x46fdbcb4ea4eadad5615cdaa17d67f783e01e21149ce2b27de497600b4cd8f4e )
  </k>
  <stack>
    #Packed ( "abcdefg" )
  </stack>
  <paramtype>
    string .AnnotationList
  </paramtype>
  <storagetype>
    bytes .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
